en/tidb-toolkit.md

```diff
@@ -200,8 +200,6 @@ helm search repo pingcap
 
 ```
 NAME                    CHART VERSION   APP VERSION     DESCRIPTION
-pingcap/tidb-backup     v1.6.0                          A Helm chart for TiDB Backup or Restore
-pingcap/tidb-cluster    v1.6.0                          A Helm chart for TiDB Cluster
```

Comment:
@srstack Could you please confirm whether lines 180-181 need to be removed?
https://github.com/pingcap/docs-tidb-operator/blob/9842802ced6b75fd223af049874add8ec892a848/en/tidb-toolkit.md?plain=1#L180-L181

Reply:
L180-L181 should be removed